openlcs/packages/views.py

```diff
@@ -933,12 +934,55 @@ def post(self, request, *args, **kwargs):
                               "source_scan_flag": None})
 
 
+class ComponentFilter(django_filters.FilterSet):
+
+    name = django_filters.CharFilter(field_name='name', lookup_expr='iexact')
+    type = django_filters.CharFilter(
+            field_name='type', lookup_expr='icontains')
+    version = django_filters.CharFilter(
+            field_name='version', lookup_expr='iexact')
+    release = django_filters.CharFilter(
+            field_name='release', lookup_expr='iexact')
+    arch = django_filters.CharFilter(field_name='arch', lookup_expr='iexact')
+    purl = django_filters.CharFilter(
+            field_name='purl', lookup_expr='icontains')
+    uuid = django_filters.CharFilter(field_name='uuid', lookup_expr='iexact')
+    summary_license = django_filters.CharFilter(
+            field_name='summary_license', lookup_expr='iexact')
+    is_source = django_filters.BooleanFilter(field_name='is_source')
+    synced = django_filters.BooleanFilter(field_name='synced')
+    source__name = django_filters.CharFilter(lookup_expr='iexact')
+    source__state = django_filters.NumberFilter()
+    source__archive_type = django_filters.CharFilter(lookup_expr='iexact')
+    source__scan_flag = django_filters.CharFilter(lookup_expr='icontains')
+
+    class Meta:
+        model = Component
+
+        fields = (
```

Comment:
I don't think each field needs a new line.

Reply:
Do you update your code to this PR, I don't find it

Reply:
submitted.